app/javascript/packs/reducers/portfolios.js

```diff
@@ -35,8 +35,9 @@ function fetchPortfolios(state, action) {
 export default function portfolios(state = {}, action) {
   if (!action) return state;
   switch (action.type) {
-    case FETCH_PORTFOLIOS: return fetchPortfolios(state, action);
-  default:
-    return state;
+    case FETCH_PORTFOLIOS:
+      return fetchPortfolios(state, action);
```

Comment:
please put this back, we are using inline case statements for reducers in many of the reducer files